sherpa-onnx/python/csrc/online-model-config.cc

```diff
@@ -28,7 +28,7 @@ void PybindOnlineModelConfig(py::module *m) {
                     const OnlineParaformerModelConfig &,
                     const OnlineWenetCtcModelConfig &,
                     const OnlineZipformer2CtcModelConfig &, const std::string &,
-                    int32_t, bool, const std::string &, const std::string &>(),
+                    int32_t,int32_t, bool, const std::string &, const std::string &>(),
```

Comment:
Please also add
```
py::arg("warm_up") = 0
```
below.

Reply:
Yes, added this.
updating a new commit with fixes for the build too.